src/dev.rs

```diff
@@ -613,7 +713,7 @@ impl<F: FieldExt> MockProver<F> {
                             .iter()
                             .map(|c| load(c, input_row))
                             .collect();
-                        let lookup_passes = (0..n)
+                        let lookup_passes = (0..n as usize)
```

Comment:
Shouldn't this also be `usable_rows`?

Reply:
Are input expressions on usable rows actually prevented from being equal to table expressions on unusable rows?

Reply:
Ok, so I've looked at the lookup argument again, and both the input and table expressions are only permuted over the usable rows. So this indeed should be `usable_rows`.